### ISSUE TITLE
Option to open twitter URL's in Tweetian

### ISSUE THREAD

**Issue by anandrkris** [veskuh/Tweetian#179]
It will be nice if twitter\* links in Tweetian have an option to open in Tweetian itself instead of browser. For instance, based on a good tweet i may want to follow the person.

Note: Not sure if this info is helpful anyway, but in the latest Jolla Together app, as mentioned in [release notes](https://github.com/tace/jolla2gether/wiki/ReleaseNotes)

> > TJC links can be now opened with jolla2gether app

https://github.com/tace/jolla2gether/commit/64a8cc6fb4a34f04e7d87165f49f027600b74efb) - Added feature: Opening TJC links with jolla2gether app

I also saw your comment in a related request to extend API in browser repo.
https://github.com/sailfishos/sailfish-browser/issues/34

Also, can Tweetian support webview to display page inside tweetian itself like Jolla Together and Tidings app?

**Comment by veskuh** [comment]
Opening links to tweets inside Tweetian is a good idea, and should be doable.

I don't like idea of making in-app webview. Making it good is a lot of work, and still platform's browser will be bettter for browsing. I personally don't like it in any of the apps or OSes that I have, so I'm not going to do it for Tweetian.

If somebody wants to contribute in-app browser I will merge if the feature is configurable and by default off.

**Comment by anandrkris** [comment]
Thanks for responding. 
Hmm, appreciate your view on the in-app web view particularly as the native browser is gaining enhancements.

**Comment by BonoNL** [comment]
Do you mean links such as cards.twitter.com and so on?

**Comment by anandrkris** [comment]
@BonoNL What I had in mind was - For instance, someone sharing a twitter status link...say, their own previous post or some other post

**Comment by bjorkstromm** [comment]
Is a "Open In Browser" pulley menu item in Tweet view sufficient? I could take a look at it.

**Comment by veskuh** [comment]
No. Tweet view already has 4 items in pulley menu. That is maximum for good usability. Clicking on a link should open it. If its link to tweet it should be displayed as tweet in tweetian. 

 For the in- app browser idea, which I still don't like, the feature would be enabled in settings and if setting is turned on links would be opened in app web view, but otherwise on browser.

**Comment by benaryorg** [comment]
A further problem then would be that, as most of us might know, Tweetian has it's bugs, for example the loading of a conversation or GIFs which are not playing in Tweetian.

In this case I want a link to open in the browser.
So I really need that option.
You could make the timestamp in the tweet view a link to the tweet and then redirect all Twitter-link clicks to a new Tweetian page.

**Comment by benaryorg** [comment]
Also, with the new quoting feature this got really important.
